synapse/handlers/message.py

```diff
@@ -378,7 +378,7 @@ def create_event(self, requester, event_dict, token_id=None, txn_id=None,
             if txn_id is not None:
                 builder.internal_metadata.txn_id = txn_id
 
-            event, context = yield self._create_new_client_event(
+            event, context = yield self.create_new_client_event(
```

Comment:
yeah, you see you made the mistake of expecting the thing that is named like a private field to not be gut-wrenched across the codebase.

Reply:
oh, no you didn't. you just did the other changes in a commit called "Update copyright" which ... I didn't expect.

Reply:
Oh, blast, I think that might have been vscode doing the changes without saving the open files. Sigh.

Reply:
I have unfucked it now.